libs/domains/organizations/data-access/src/lib/domains-organizations-data-access.ts

```diff
@@ -18,6 +25,58 @@ export const organizations = createQueryKeys('organizations', {
       return response.data
     },
   }),
+  gitTokens: (organizationdId) => ({
+    queryKey: [organizationdId],
+    async queryFn() {
+      const response = await organizationApi.listOrganizationGitTokens(organizationdId)
+      return response.data.results
+    },
+  }),
+  authProviders: (organizationId) => ({
+    queryKey: [organizationId],
+    async queryFn() {
+      const response = await gitApi.getOrganizationGitProviderAccount(organizationId)
+      return response.data
```

Comment:
```suggestion
      return response.data.results
```

Reply:
The API response returns the data on the `response.data`, not on the `response.data.results` I'll see with back-end if it's normal